springfield/firefox/templates/firefox/ai/waitlist.html

```diff
@@ -0,0 +1,146 @@
+{% endblock %}
+
+{% block content %}
+<nav class="c-navigation" aria-label="{{ ftl('navigation-landmark-label') }}">
+    <a href="{{ url('firefox') }}" data-link-text="firefox home icon" data-link-position="nav">
+      <img src="{{ static('img/logos/firefox/logo-word-hor.svg') }}" alt="{{ ftl('navigation-firefox-home') }}" width="120" height="40">
+    </a>
+  </nav>
+<main class="c-main">
+  <div id="main-content" class="c-main-content">
+    <h1 class="c-pill"><span class="c-pill-tag">Coming Soon</span> <span>AI Window</span></h1>
+    <h2 class="c-heading">AI Window: Built for choice &amp; control</h2>
+    <div class="c-body">
+      <p><strong>The web is changing, and sitting it out doesn’t help anyone.</strong></p>
+      <p>We were founded under the mission that the web should be open, fast, secure and people first. As AI becomes a larger part of the web, we see a clear role: to shape it in ways that support choice and keep the internet open and accessible to all. With AI Window we’ll do this through:</p>
+      <ul class="mzp-u-list-styled">
+        <li>A fully opt-in experience</li>
+        <li>Features that protect your choice</li>
+        <li>More transparency around how your data is used</li>
+      </ul>
+      <p>Our success has always been driven by a community of users and developers. Let’s explore how AI can serve the open web together.</p>
+      <p>Sign up to receive updates and to be among the first to try it</p>
+    </div>
+    <!-- forked from cms/newsletter_form -->
+    <form data-testid="newsletter-form" id="newsletter-form" action="{{ settings.BASKET_SUBSCRIBE_URL }}" method="post" novalidate>
+      <div hidden="">
+        <div id="id_newsletters">
+          <div>
+            <label for="id_newsletters_0"><input type="checkbox" name="newsletters" value="{{ newsletter_id }}" id="id_newsletters_0" checked="">
+  Firefox</label>
+          </div>
+        </div>
+      </div>
+      <input type="hidden" name="source_url" value="{{ request.build_absolute_uri() }}">
+      <input type="hidden" name="lang" id="id_lang" value="en">
+      <label class="visually-hidden" for="newsletter-email">{{ email_label|d(ftl('newsletter-form-your-email-address'), true) }}</label>
+      <div class="c-input-wrapper">
+        <input data-testid="newsletter-email-input" id="newsletter-email" class="mzp-js-email-field" type="email" name="email" placeholder="email@example.com" autocomplete="email" class="c-input" />
+        <button data-testid="newsletter-submit-button" id="newsletter-submit" type="submit" class="c-submit">
+          <span class="c-submit-text">Join the waitlist</span>
+          <svg class="c-submit-loading" fill="currentColor" width="24" height="24" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg">
+            <title>Sending</title>
+            <style>.s1{animation:s .8s linear infinite;animation-delay:-.8s}.s2{animation-delay:-.65s}.s3{animation-delay:-.5s}@keyframes s{93.75%,100%{opacity:.2}}</style>
```

Comment:
CSP blocks this inline style. This kind of SVG only works linked as an image (it's available as [loading.svg](https://github.com/mozmeao/springfield/blob/main/media/img/icons/loading.svg) in icons.)

Reply:
(e.g. if this output, with just "bubbles", not animating, is acceptable, then just the `<style/>` block can be removed to not submit the violations, and left as-is for this page…)

So the easy thing here would be to just externalize that to an img, but it probably will only start loading once shown, it would have to be preloaded in some way for the intended effect. (It was really added to be used as a CSS background and swapped in-place ~ however I'm not familiar with the FDS markup to be looking into where to place it for the button to have it preloaded best.)